openmdao/drivers/test/test_pyoptsparse.py

```diff
@@ -131,8 +130,8 @@ def test_simple_paraboloid_equality(self):
         prob.run()
 
         # Minimum should be at (7.166667, -7.833334)
-        assert_rel_error(self, prob['x'], 7.16667, 1e-6)
-        assert_rel_error(self, prob['y'], -7.833334, 1e-6)
+        assert_rel_error(self, prob['x'], 7.16667, 1e-3)
```

Comment:
Could also set the SLSQP tolerance a little tighter (requires setting a key into the pyopt_options dict)